File: introduction/ITP1_3_C-Swapping Two Numbers.js
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function(chunk) {
  var lines = chunk.toString().trim().split('\n');
  for (var i = 0; i < lines.length; i++) {
    var line = lines[i];
    if (line == '0 0') {
      break;
    }
    line = line.split(' ').sort(function(a, b) {
      return a - b;
    });
    console.log(line.join(' '));
  }
});

File: introduction/ITP1_3_C-Swapping Two Numbers.rs
use std::io::*;
fn main() {
  let stdin = stdin();
  for line in stdin.lock().lines() {
    let mut l: Vec<_> = line.unwrap().split_whitespace().map(|s| s.parse::<i32>().unwrap()).collect();
    if l == [0, 0] { break; }
    l.sort();
    println!("{} {}", l[0], l[1]);
  }
}
